src/lib/backend.c

```diff
@@ -25,12 +42,14 @@ static bool esysdb_init = false;
 CK_RV backend_init(void) {
     LOGV("Initializing backends");
 
-    bool is_fapi_preview = is_fapi_preview_enabled();
+    if (get_backend() == backend_error) {
```

Comment:
I forgot to mention this, no functions inside conditionals unless it makes the code simpler (like an fgetline loop). Here and throughout.

Reply:
done